README.md

```diff
@@ -22,26 +23,29 @@ chmod -R o+w input_files
 ## Execution
 ```
 docker-compose -f docker-compose-CeleryExecutor.yml up -d
-Open Airflow UI: http://<server_ip>:8080/admin/
-Toggle switch to enable/start the DAG
+
+Open Airflow UI: http://<docker_host_ip>:8080/admin/
+
+Toggle switch from "Off" to "On" to enable the SandboxSubmision DAG
+
 ```
 
 ## Test Run:
 ```
 Open commmand prompt or terminal and run:
 docker container ps
 
-Note the CONTAINER_ID of the container running airflow webserver
+Note the CONTAINER_ID of the container running airflow webserver. The airfow webserver will have "/entrypoint.sh webserver" in the COMMAND column of the docker container ps output.
 
 Execute this command below to trigger the DAG with the provided sample study:
 docker exec -it <REPLACE_WITH_CONTAINER_ID_FROM_ABOVE> python /usr/local/airflow/.local/bin/run_qc.py --host_port localhost:8080 -r input_files -o output_files -m input_files/user_study/metadata.json -s user_study
 
-Open Airflow UI: http://<server_ip>:8080/admin/ to check status of DAG run
+Open Airflow UI: http://<docker_host_ip>:8080/admin/airflow/graph?dag_id=SandboxSubmission/ to check status of DAG run
```

Comment:
The trailing / in the URL doesn't work.

I guess it would help to display this URL as part the output of our command line script output